corehq/apps/dump_reload/management/commands/load_domain_data.py

```diff
@@ -68,11 +68,14 @@ def add_arguments(self, parser):
                                  " against a CouchDB 'doc_type' or Django model name: 'app_label.ModelName'."
                                  "Use 'print_domain_stats' command to get a list of available types.")
         parser.add_argument('--json-output', action="store_true", help="Produce JSON output for use in tests")
+        parser.add_argument('--chunksize', action="store_true", default=None,
```

Comment:
action is incorrect and it must be an int
```suggestion
        parser.add_argument('--chunksize', type=int, default=None,
```